### PR TITLE
chore: api spec update

### PR DESCRIPTION
**Pull request by a-m-dev**
for PT

### CODE REVIEW

**Comment by a-m-dev** [comment]
why api-governance step keeps failing? any idea about it?

**Comment by cbt1** [comment]
> yes, you have updated the api-specification without changing the version. You should only re-generate the api specs when making a release

My bad, I told him to update it.

File: src/qae/initial-properties.d.ts [hunk 1]
```diff
@@ -1,32 +1,35 @@
-declare module "initial-properties";
+import { Component } from "../types/QIX";
```
Comment (cbt1) [inline src/qae/initial-properties.d.ts:1]: `Component` does not exist. You need to add it

Comment (a-m-dev) [reply src/qae/initial-properties.d.ts:1]: I actually followed what we do in table side: https://github.com/qlik-oss/sn-table/blob/main/src/qae/object-properties.d.ts#L1

is that one correct or should I declare something like: 
```ts
declare const RowHeight {
  linesCount: number;
}

declare const Component {
  key: "general" | "theme";
  rowHeight?: RowHeight;
}
```

Comment (a-m-dev) [reply src/qae/initial-properties.d.ts:1]: Actually, any declarations in `*.d.ts` file is only for our local development, for our project and internal usage of typescript to not make it confuse with types. 
So the thing is `Component` interface is already declared inside of `types/QIX` and if we are using this as internal (more) type interference of typescript, it makes sense to give it what it needs (import the `Component` interface and use it in `properties.component`)

Comment (cbt1) [reply src/qae/initial-properties.d.ts:1]: Looks like I was on a branch that was not up-to-date with `main` so I did not have `Component` defined

**Review by cbt1 — APPROVED** [review]

**Pull request merged**